File: encapsulation.java
public class Container{
    String liquid;

    public String getLiquid(){
        return liquid;
    }

    public void setLiquid(String liq){
        this.liquid = liq
    }
}

Container c = new Container();
Container m = new Container();

c.setLiquid("Water");
m.setLiquid("Milk");

String liqd = c.getLiquid();
System.out.println(liqd);
System.out.println(m.getLiquid());

File: encapsulation.py
class Container:
    def getLiquid(self):
        return self.liquid 
    
    def setLiquid(self,liq):
        self.liquid = liq
    
    def __init__(self):
        self.liquid = None
    

c = Container()
m = Container()

c.setLiquid("Water")
m.setLiquid("Milk")

liqd = c.getLiquid()
print(liqd)
print(m.getLiquid())



